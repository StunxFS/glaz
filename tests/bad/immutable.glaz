fn main() {
  let x = 100;
  x = 10;
}
