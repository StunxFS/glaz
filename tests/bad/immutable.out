tests/bad/immutable.glaz:3:4: error: cannot assign twice to immutable variable `x`
     | 
   3 |   x = 10;
     |   ^
     = note: declare it with `mut` to make it mutable: `mut x`
tests/bad/immutable.glaz:2:8: warning: variable `x` declared but never used
     | 
   2 |   let x = 100;
     |       ^
     = help: if this is intentional, prefix it with an underscore: `_x`
     = help: or use the blank identifier (`_`)
glaz error: could not compile `immutable`, aborting due to previous error.
